### PR TITLE
Pt 157551442 split block on type

### PR DESCRIPTION
**Pull request by gorillainduction**
https://www.pivotaltracker.com/story/show/157551442

### CODE REVIEW

File: deployment/DB_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-79
+80
```
Comment (lucafavatella) [inline deployment/DB_VERSION]: Master seems [already at 80](https://github.com/aeternity/epoch/blob/452fcbd65e9db42f81bf222325023e88b2eaad03/deployment/DB_VERSION#L1) - blocking so we remember.

BTW there will be conflicts e.g. https://github.com/aeternity/epoch/pull/1470

**Comment by lucafavatella** [comment]
This seems will conflict with #1467 too cc @fabiankrol @jur0

**Comment by gorillainduction** [comment]
@lucafavatella Rebased on master and resolved conflict.
I think the conflict with #1467 will be trivial to resolve.
(cc @fabiankrol @jur0 )

File: apps/aecore/src/aec_blocks.erl [hunk 2]
```diff
@@ -1,7 +1,10 @@
+%%% -*- erlang-indent-level:4; indent-tabs-mode: nil -*-
```
Comment (lucafavatella) [inline apps/aecore/src/aec_blocks.erl]: We may consider slowly moving away from these using [editorconfig](https://github.com/aeternity/epoch/blob/78269fd936122b74fd00be617320ad4758b698b0/.editorconfig#L4-L5).

File: apps/aecore/src/aec_subscribe.erl [hunk 3]
```diff
@@ -82,7 +82,10 @@ handle_info({gproc_ps_event, Event = micro_block_created, #{ info := Block }}, S
 
 notify_subscribers(top_changed = Event, Block, State = #state{ subscribed = Subs }) ->
     notify_chain_subscribers(Event, Block, Subs),
-    notify_tx_subscribers(aec_blocks:txs(Block), Subs),
+    case aec_blocks:type(Block) of
+        micro -> notify_tx_subscribers(aec_blocks:txs(Block), Subs);
+        key   -> ok
```
Comment (lucafavatella) [inline apps/aecore/src/aec_subscribe.erl]: I see this does not change behaviour (the line above calls `notify_chain_subscribers`). Ok.

**Pull request merged**